make-silverlsp-vsix

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Utility to clean-rebuild and package the Silver language server
+# and VS Code extension.
+
+set -eu
+
+SILVERLSP_VSIX="silverlsp-latest.vsix"
+
+if [ $0 != "./make-silverlsp-vsix" ]; then
+  echo "Run as ./make-silverlsp-vsix"
+  exit 1
+fi
```

Comment:
any reason not to do `cd "$(dirname "${BASH_SOURCE[0]}")"` instead?

Reply:
Ehh I just copied that bit from all the other top-level utility scripts.  I could change it I guess?